src/hooks/useUploadsManager/store.tsx

```diff
@@ -0,0 +1,71 @@
+import create from 'zustand'
+import { persist } from 'zustand/middleware'
+
+import { AssetUpload, AssetUploadStatus, UploadsProgressRecord } from './types'
+
+type AssetFile = {
+  contentId: string
+  blob: File | Blob
+}
+
+type UploadStoreState = {
+  uploadsState: AssetUpload[]
+  addAsset: (asset: AssetUpload) => void
+  setUploadsProgress: (contentId: string, progress: number) => void
+  updateAsset: (contentId: string, lastStatus: AssetUploadStatus) => void
+  removeAsset: (contentId: string) => void
+  uploadsProgress: UploadsProgressRecord
+  notifications: {
+    uploading: number
+    uploaded: number
+  }
```

Comment:
What do we need those for? Seems `useStartFileUpload` already has its own notification counts?